__tests__/app-tests/app.get.test.js

```diff
@@ -546,3 +546,44 @@ describe("GET /api/articles?sort_by=value&order=value", () => {
     });
   });
 });
+
+describe("GET /api/articles?topic=value", () => {
```

Comment:
This can simply be combined with the previous describe for the same base endpoint